r/content/creating_arrow_objects.Rmd

```diff
@@ -0,0 +1,86 @@
+# Creating Arrow Objects
+
+## Build an Arrow Table from native language types
+
+### Manually create a table from an R object
+
+You may want to convert an existing data frame in R to an Arrow Table object.
+
+```{r, table_create}
+# Create an example data frame
+my_tibble <- tibble::tibble(group = c("A", "B", "C"), score = c(99, 97, 99))
+# Convert to Arrow Table
+my_table <- Table$create(my_tibble)
+# View table
+my_table
+```
+```{r, test_table_create, opts.label = "test"}
+test_that("table_create works as expected", {
+  expect_s3_class(my_table, "Table")
+  expect_identical(dplyr::collect(my_table), my_tibble)
+})
+```
+#### View the contents of an Arrow Table
+
+You can view the contents of an Arrow table using `dplyr::collect()`
```

Comment:
For consistency, I think it's best to always capitalize "Table" when referring to an Arrow Table, and always render "RecordBatch" as one word in upper camel case.
```suggestion
You can view the contents of an Arrow Table using `dplyr::collect()`
```